scala/scala-impl/src/org/jetbrains/plugins/scala/project/Versions.scala

```diff
@@ -66,15 +67,16 @@ object Versions  {
 
   private def loadVersionsForHydra() = {
     val entity = Entity.Hydra
+    val repoUrl = HydraRepository.getRepositoryUrl + entity.url
 
     def downloadHydraVersions(url: String): Seq[String] =
       loadVersionsFrom(url, { case entity.pattern(number) => number }).getOrElse(entity.hardcodedVersions).map(Version(_))
         .filter(_ >= entity.minVersion).map(_.presentation)
 
-    loadVersionsFrom(entity.url, {
+    loadVersionsFrom(repoUrl, {
       case entity.pattern(number) => number
     }).map { versions =>
-      versions.flatMap(version => downloadHydraVersions(s"""${entity.url}$version/""")).distinct
+      versions.flatMap(version => downloadHydraVersions(s"""$repoUrl$version/""")).distinct
```

Comment:
Aren't we missing a `/` between `$repoUrl` and `$version`?

Reply:
No, because here is how I create it `val repoUrl = HydraRepository.getRepositoryUrl + entity.url` and the `entity.url` is `/ivy-releases/com.triplequote/`